js/flightlog_parser.js

```diff
@@ -483,7 +490,14 @@ var FlightLogParser = function(logData) {
                 that.sysConfig.rcYawRate = parseInt(fieldValue, 10);
                 break
             case "rc_expo":
-                that.sysConfig.rcExpo = parseInt(fieldValue, 10);
+                if(fieldValue.match(/.*,.*/)!=null) {
```

Comment:
What does `parseCommaSeparatedString()` return if it's not a comma separated string?

It's more robust to use the `parseCommaSeparatedString()` function to check if it matches (and return null if not), than to do the checking here and the parsing in the function.

Reply:
I've reused that check from somewhere else in the code.

```
function parseCommaSeparatedString(string, length) {
    /***
     * Parse a comma separated string for individual values.
     *
     * string               is the comma separated string to parse
     * length (optional)    the returned array will be forced to be this long; extra fields will be discarded,
     *                      missing fields will be padded. if length is not specified, then array will be auto
     *                      sized.
     *
     * returns              if the string does not contain a comma, then the first integer/float/string is returned
     *                      else an Array is returned containing all the values up to the length (if specified)
     ***/
```

The result of `parseCommaSeparatedString()` is never `null`, it always returns something. I guess that is why the check done separately.

I can make this nicer to do the check in function instead.